corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowserMain.java

```diff
@@ -38,14 +39,16 @@
         + "Options:\n"
         + "--host=<host>   Hostname\n"
         + "--port=<port>   Port\n"
-        + "--operation=<listTables|infoTable|showTable|dropTable> Operation\n"
+        + "--operation=<listTables|infoTable|showTable|dropTable|loadTable> Operation\n"
         + "--namespace=<namespace>   Namespace\n"
         + "--tablename=<tablename>   Table Name\n"
         + "--keystore=<keystore_file> KeyStore File\n"
         + "--ks_password=<keystore_password> KeyStore Password\n"
         + "--truststore=<truststore_file> TrustStore File\n"
         + "--truststore_password=<truststore_password> Truststore Password\n"
         + "--diskPath=<path to temp folder for large tables> Path to Temp Dir\n"
+        + "--numItems=<number of items to load> Number of items to load for loadTable\n"
```

Comment:
looks like we dont use these parameters.  Can they be removed?

Reply:
it is used but not as what it looks like on first glance - will also add an option to insert multiple items instead of same key multiple times